portal-impl/src/com/liferay/portlet/announcements/service.xml

```diff
@@ -106,7 +106,7 @@
 		<!-- Audit fields -->
 
 		<column name="companyId" type="long" />
-		<column name="userId" type="long" />
+		<column name="userId" type="long" uad-anonymize-field-name="userId" />
```

Comment:
@julschong why this is changed? No generated code related to this change.

Reply:
@tinatian Hi Tina, This is related to BaseAnnouncementsFlagUADAnonymizer.

Reply:
If this line is not added, BaseAnnouncementsFlagUADAnonymizer does not generate at all